content/applications/marketing/events/overview.rst

```diff
@@ -0,0 +1,12 @@
+:nosearch:
+
+========
+Overview
+========
+
+.. toctree::
+   :titlesonly:
+
+   overview/create
+   overview/selling_tickets
```

Comment:
Optional, if you want the file to have the same name as the page title.
```suggestion
   overview/sell_tickets
```
If you do change that, don't forget to rename the folder too.

Reply:
Will do for all files/folders 👍